### ISSUE TITLE
UncaughtException: TypeError: Cannot read property 'getProp' of undefined. Restarting Server...

### ISSUE THREAD

**Issue by axe-z** [tststs/atom-ternjs#328]
I get this while scrolling, it's really random... not sure what is happening ...

**Comment by TroublemakerSt** [comment]
Same! Really annoying

**Comment by tststs** [comment]
Without code, example repo where the issue happens or a stack trace i'm not able to anything.

**Comment by xiersa** [comment]
Me too! Too annoying...

**Comment by AlanOzil** [comment]
I tried to disable "atom-ternjs" package.It was fixed

**Comment by Altamill** [comment]
I found ternjs triggered getProp error only when the cursor was positioned in the Vue code ")"

**Comment by barone-dev** [comment]
It started here, really annoying, i'm coding Vue.

**Comment by CristianaPereira** [comment]
Same here. Using Vue too.

**Comment by aielloine** [comment]
Same here but not use vue

**Comment by Xhyen** [comment]
> Same here but not use vue

Do you use atom-ternjs? try disable it

**Comment by Cobatkao** [comment]
same here, coding .vue, it really annoying

**Comment by ednilsonamaral** [comment]
any solution?

**Comment by torlun** [comment]
Same here. .vue files only.

**Comment by unoscar** [comment]
Same Here, not using Vue

**Comment by capjuancode** [comment]
Same Here, with Vue

**Comment by IanRr** [comment]
also having this issue in Riot -- guess it's time to disable tern and find out how much I use it

**Comment by Zireael** [comment]
Video showing the issue (editing Vue components/JS):

https://streamable.com/vw1qn

**Comment by fonziemedia** [comment]
Same here, not using Vue

**Comment by axelrindle** [comment]
I receive this error when modifying a **.blade.php** file... could be of some interest :man_shrugging:

**Comment by Esskimo** [comment]
Same here. Got this error when using Handlebars or Vue.

**Comment by CalvinD3v** [comment]
I just disabled a package called **atom-ternjs** and it stopped  popping out .

**Comment by axelrindle** [comment]
@CalvinD3v disabling the package causing the issue is not an option here :thinking:

**Comment by CalvinD3v** [comment]
@axelrindle you actually right it's back again for some reasons.

**Comment by rubenish** [comment]
Hi, i get the same error of "UncaughtException: TypeError: Cannot read property 'getProp' of undefined. " when i try to change the code of a HTML project, but when i modify the code of a JS File i don't get the error...

**Comment by DevAnuragGarg** [comment]
Same here.

**Comment by pixelpaulaus** [comment]
very anoying

**Comment by Gaffen** [comment]
Also getting the issue when editing pretty much any file

**Comment by spookyvert** [comment]
I'm getting it when using react on Atom

**Comment by sebmor** [comment]
Same here, very annoying

**Comment by spermspace** [comment]
off ternjs )))

**Comment by ruben-ksdm** [comment]
Same here... Get this error just when editing .tmpl files (HTML).

**Comment by colorwebdesigner** [comment]
Same here while editing html files

**Comment by EidarousDev** [comment]
Disabling ternjs package seems to be a temporary solution.

**Comment by arstrel** [comment]
Same problem when editing JS inside HTML file.
UncaughtException: TypeError: Cannot read property 'getProp' of undefined. Restarting Server...

Any solution  yet? Besides disabling the package?

**Comment by vpiskunov** [comment]
Still happening - 2 years later... Just started happening a month or two ago... any fix?

**Comment by arstrel** [comment]
No, I have not found a solution. I switched to VSCode a while ago. Best
thing ever

**Comment by last-partizan** [comment]
Looks like ternjs error.

And i made [pull request to fix it](https://github.com/ternjs/tern/pull/1023).

**Comment by cloudop** [comment]
> Looks like ternjs error.
> 
> And i made [pull request to fix it](https://github.com/ternjs/tern/pull/1023).

Thanks!!!

**Comment by last-partizan** [comment]
Crash should be fixed in next minor release of tern, but atom-ternjs should not send full vue file to tern. It should send only javascript part.

**Comment by qianyinghuanmie** [comment]
same here，I will switch to VSCode。

**Comment by Steve-Hsu** [comment]
same here, I only code HTML.

**Comment by UziTech** [comment]
@tststs according to https://github.com/ternjs/tern/pull/1023#issuecomment-509675613 the issue is that atom-ternjs sends more than just javascript to ternjs.

**Comment by GreatTyrion** [comment]
I am using node.js and ejs in Atom. The issue was removed when I updated four packages: **atom-ternjs**, linter, linter-jshint, linter-ui-default.
